src/components/analytics/GoogleAnalyticsReporter.tsx

```diff
@@ -21,6 +22,11 @@ export default function GoogleAnalyticsReporter({ location: { pathname, search }
     getCLS(reportWebVitals)
   }, [])
 
+  const { chainId } = useActiveWeb3React()
+  useEffect(() => {
+    ReactGA.set({ ['Chain ID']: chainId ?? 1 })
```

Comment:
Should we use a chainId of `0` for default/unconnected?

Reply:
1 (mainnet) is the default chainId and rpc connection.

Reply:
w/ `useActiveWeb3React` "active" means the user's wallet connection. If they don't have a connection to the application, we use a fallback network connector that only uses data from mainnet.

Reply:
If a user is connected to mainnet, it will be `1`, but if a user doesn't connect a wallet, it will still be `1` from the fallback connection. AFAIK we don't allow switching from mainnet when using the fallback. Could we use `0` to denote "mainnet with fallback", as opposed to "mainnet with connected user wallet"?

Reply:
@willhennessy Would you prefer 0?

Reply:
Will says 0 works, will add.